### PR TITLE
Add vendor for BellSoft Liberica NIK

### PR DESCRIPTION
**Pull request by eddumelendez**

### CODE REVIEW

File: app/controllers/JavaListController.scala [hunk 1]
```diff
@@ -72,6 +72,7 @@ class JavaListController @Inject() (versionRepo: VersionsRepository, cc: Control
     "amzn"    -> "Amazon",
     "grl"     -> "GraalVM",
     "librca"  -> "BellSoft",
+    "nik"     -> "NativeImageKit",
```
Comment (marc0der) [inline app/controllers/JavaListController.scala:75]: Just a thought, but how about calling it `Bellsoft NIK` or `Liberika NIK`?

**Comment by marc0der** [comment]
I'm thinking of changing a lot of these soon. For instance, people call the Amazon variant Corretto. The vendor string should probably not be taken so literally, and we should be using the name that everybody knows. WDYT?

File: app/controllers/JavaListController.scala (hunk 1 above)
Comment (eddumelendez) [reply app/controllers/JavaListController.scala:75]: agree! just though there is a limitation of string length

**Comment by eddumelendez** [comment]
sounds good to me. I think corretto is the only one in the list which we call amzn

File: app/controllers/JavaListController.scala (hunk 1 above)
Comment (marc0der) [reply app/controllers/JavaListController.scala:75]: `NativeImageKit` is two characters longer than `Liberica NIK`, so that should be okay, right?

Comment (eddumelendez) [reply app/controllers/JavaListController.scala]: updated!

**Review by marc0der — APPROVED** [review]

**Comment by eddumelendez** [comment]
not sure why is failing

**Pull request merged**